src/man/sssd-idp.5.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DocBook V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<reference>
+<title>SSSD Manual pages</title>
+<refentry>
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="include/upstream.xml" />
+
+    <refmeta>
+        <refentrytitle>sssd-idp</refentrytitle>
+        <manvolnum>5</manvolnum>
+        <refmiscinfo class="manual">File Formats and Conventions</refmiscinfo>
+    </refmeta>
+
+    <refnamediv id='name'>
+        <refname>sssd-idp</refname>
+        <refpurpose>SSSD IdP provider</refpurpose>
+    </refnamediv>
+
+    <refsect1 id='description'>
+        <title>DESCRIPTION</title>
+        <para>
+            This manual page describes the configuration of the IdP provider
+            for
+            <citerefentry>
+                <refentrytitle>sssd</refentrytitle>
+                <manvolnum>8</manvolnum>
+            </citerefentry>.
+            For a detailed syntax reference, refer to the <quote>FILE FORMAT</quote> section of the
+            <citerefentry>
+                <refentrytitle>sssd.conf</refentrytitle>
+                <manvolnum>5</manvolnum>
+            </citerefentry> manual page.
+        </para>
+        <para>
+            The IdP provider is a back end used to connect to an OAuth 2.0 and
+            REST based identity provider (IdP). If the IdP follows the OpenID
+            Connect (OIDC) standard the configuration might be simpler but
+            other providers are support as well. Since products might have
+            individual implementation of the REST API for looking up user and
+            group attributes dedecated code might be required, see the
+            <quote>idp_type</quote> option for details.
+        </para>
+    </refsect1>
+
+    <refsect1 id='configuration-options'>
+        <title>CONFIGURATION OPTIONS</title>
+        <para>Refer to the section <quote>DOMAIN SECTIONS</quote> of the
+            <citerefentry>
+                <refentrytitle>sssd.conf</refentrytitle>
+                <manvolnum>5</manvolnum>
+            </citerefentry> manual page for details on the configuration of an SSSD domain.
+            <variablelist>
+                <varlistentry>
+                    <term>idp_type (string)</term>
+                    <listitem>
+                        <para>
+                            Specifies the IdP product. Currently Entra ID
+                            and Keycloak are supported. Entra ID is used as
+                            default.
+                        </para>
+                        <para>
+                            Default: Not set (Entra ID)
```

Comment:
Do we really need a default here? Wouldn't it be better if this option was always set? You need to set other options anyway.

Reply:
It might be nice to state which options are required... I assume all of them except timeout?

Reply:
I updated the man page indication the required options and made `idp_type` required as well.